services/search/src/search/routes/filter.py

```diff
@@ -21,7 +21,7 @@
     get_request_parameter_length,
     get_request_parameter_offset,
 )
-from libapi.response import ROW_IDX_COLUMN, create_response
```

Comment:
what was it used for? why do we remove it?

Reply:
It was used in execute_filter_query (Line 188). Since all split-duckdb-index have been computed, all cache entries now have the features as part of the content, so, no need to add it to the list of columns.